apis/swagger.yml

```diff
@@ -3906,6 +3906,37 @@ definitions:
       CheckpointName:
         type: "string"
 
+  ContainerCommitOptions:
```

Comment:
please also add the API routes for this API endpoint.  And I think this is not the first time to remind you. 😄